From 9b11d11ef63792bf3dae239d3e03df2027c85bcf Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Mon, 21 Aug 2017 11:43:07 -0400
Subject: [PATCH] PUBAPI-1426 sdc-cloudapi incorrectly handles machine create
 test failure Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved
 by: Marsell Kukuljevic <marsell@joyent.com>

---
 test/machines.70.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/machines.70.test.js b/test/machines.70.test.js
index 238b6f9..3ca64cf 100644
--- a/test/machines.70.test.js
+++ b/test/machines.70.test.js
@@ -152,7 +152,7 @@ test('Wait For Running', function (t) {
 
 
 test('Get Machine,  with Firewall Enabled', function (t) {
-    if (MACHINE_UUID) {
+    if (!MACHINE_UUID) {
         return t.end();
     }
 
-- 
2.21.0

